sauna/plugins/ext/http_json.py

```diff
@@ -0,0 +1,119 @@
+from sauna.plugins import Plugin, PluginRegister
+from sauna import DependencyError
+import re
+import json
+
+my_plugin = PluginRegister('HTTP-JSON')
+
+
+@my_plugin.plugin()
+class HTTPJSON(Plugin):
```

Comment:
This class looks a lot like the `HTTP` one. Is it possible somehow to inherit from it to avoid duplicating the path that actually makes the request?